eng/Directory.Build.Data.targets

```diff
@@ -28,17 +28,13 @@
     <PropertyGroup>
       <PowerShellExe Condition=" '$(PowerShellExe)'=='' ">"%ProgramFiles%\PowerShell\6\pwsh.exe"</PowerShellExe>
       <PowerShellExe Condition="!Exists('$(PowerShellExe)')">pwsh</PowerShellExe>
-      <GetReleaseNotesScriptPath Condition=" '$(GetReleaseNotesScriptPath)'=='' ">$(MSBuildThisFileDirectory)/common/Extract-ReleaseNotes.ps1</GetReleaseNotesScriptPath>
+      <ValidateReleaseNotesScriptPath Condition=" '$(ValidateReleaseNotesScriptPath)'=='' ">$(MSBuildThisFileDirectory)scripts/Validate-ReleaseNotes.ps1</ValidateReleaseNotesScriptPath>
```

Comment:
Why isn't this just calling https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/Verify-ChangeLog.ps1

Reply:
It would allow us to stop trying to figure out the changelog path in msbuild as well.

Reply:
If we still need the path in msbuild then we can either add more parameters to the common script or figure out a way to set the value so that msbuild can pick it up.

Reply:
I modified it to use the same script. am keeping changelog path because otherwise we will need to pass other parameters like service name, and package name, it is easier to just still pass the changelog path.